src/tactic/interactive_expr.lean

```diff
@@ -390,8 +391,16 @@ tc.stateless $ λ ft, do
   let lcs := group_local_collection lcs,
   lchs ← lcs.mmap (λ lc, do
     lh ← local_c lc,
-    let ns : list (html γ) := lc.locals.map $ λ n,
-      h "span" [cn "goal-hyp b pr2", key n.local_uniq_name] [html.of_name n.local_pp_name],
+    cls ← is_class lc.type,
+    let ns := lc.locals.map (λ n,
+      let var_color : attr γ := if ¬ cls
+        then attr.style [("color", "#cc7a00")]  -- "goal-hyp"
+        else match is with
+        | some is' := if n ∈ is'
+                        then attr.style [("color", "#2aa198")]  -- "goal-hyp-inst"
+                        else attr.style [("font-style", "italic")] -- "goal-hyp-noninst"
+        | none := attr.style [("color", "#dc322f")] -- "goal-hyp-unfrozen"
+        end in h "span" [cn "goal-hyp b pr2", var_color] [html.of_name $ expr.local_pp_name n]),
```

Comment:
What do you think of just adding classes instead of adding color/font-style directly? Then all the styling can be done in css.

Reply:
The comments are supposed to be the CSS classes we want to use for these styles. Since the styles are specified in the VS Code extension, we can't actually display the variables differently at the moment, before updating the extension.

Reply:
Yeah, this is what I meant in the PR description. Sorry if that was misphrased!

Reply:
Sure, but this PR should just add those CSS classes, right? And then we can debate the colors/font-styles in the vscode-lean PR.

Reply:
But the vscode-lean PR will need to be merged first, right?

Reply:
Why? If there is no CSS corresponding to those classes then they will just be ignored.